docs/docs-beta/docs/guides/non-python.md

```diff
@@ -1,73 +1,84 @@
+- `DAGSTER_PIPES_MESSAGES` - Output context
 
-- `DAGSTER_PIPES_CONTEXT`: Input context
-- `DAGSTER_PIPES_MESSAGES`: Output context
+Create a new file with the following helper functions that read the environment variables, decode the data, and write messages back to Dagster:
 
-Create the following helper functions for reading the environment variables, decoding the data, and writing messages back to Dagster.
+<CodeExample filePath="guides/non-python/pipes-javascript-utility.js" language="javascript" />
 
-<CodeExample filePath="guides/non-python/pipes-javascript-utility.js" language="javascript" title="Utility functions to interface with Dagster Pipes." />
+Both environment variables are base64 encoded, zip compressed JSON objects. Each JSON object contains a path that indicates where to read or write data.
 
-Both environment variables are base64 encoded, zip compressed JSON objects. Each JSON object contains a _path_ that indicate where to read or write data.
+## Step 4: Integrate the utility functions into the script
 
-## Integrate the utility functions into your original script
+Using the utility functions to decode the Dagster Pipes environment variables, you can send additional parameters into the JavaScript process. You can also output more information into the asset materializations.
 
-With the utility functions to decode the Dagster Pipes environment variables, we can send additional parameters into the JavaScript process and output additional information into the asset materializations.
+Update the `tensorflow/main.js` script to:
 
-Update the `tensorflow/main.js` script to retrieve the model configuration from the Dagster context, and report an asset materialization back Dagster with model metadata.
+- Retrieve the model configuration from the Dagster context, and
+- Report an asset materialization back to Dagster with model metadata
 
-<CodeExample filePath="guides/non-python/pipes-full-featured-javascript.js" language="javascript" title="Adding a new JavaScript entrypoint for Dagster Pipes." />
+<CodeExample filePath="guides/non-python/pipes-full-featured-javascript.js" language="javascript" />
 
-## Update your Dagster asset
+## Step 5: Update the Dagster asset
 
-Finally, update your Dagster asset to pass in the model information that's used by your script.
+Finally, update your Dagster asset to pass in the model information that's used by the script:
 
-<CodeExample filePath="guides/non-python/pipes-asset-with-context.py" language="python" title="Asset using Dagster Pipes." />
+<CodeExample filePath="guides/non-python/pipes-asset-with-context.py" language="python" />
 
 ## What's next?
 
-From here, this example could be extended to support many JavaScript defined operations and many JavaScript based assets using different pairings of operations and configurations.
+From here, you can extend this example to support numerous JavaScript-defined operations and assets, using various combinations of operations and configurations.
```

Comment:
I reworded this from the original guide, but I'm not sure it's actually meaningful/helpful. Open to ideas.